include/picongpu/plugins/particleMerging/ParticleMerger.kernel

```diff
@@ -204,76 +204,76 @@ namespace particleMerging
                     auto particle = frame[linearThreadIdx];
 
                     const voronoiCellId::type voronoiCellId = particle[voronoiCellId_];
-                    if( voronoiCellId == -1 )
-                        return;
```

Comment:
I do not see a difference between both codes. The return is only exiting the lambda function and not skipping other invocations from `forEachFrame`.
Maybe I overlook something but for me both versions are identically from the code flow view.

Reply:
Yes, good point. Then maybe two changes were made at once and we got confused.